zproject/settings.py

```diff
@@ -716,6 +716,7 @@ def get_secret(key):
             'js/referral.js',
             'js/custom_markdown.js',
             'js/bot_data.js',
+            'js/pointer.js'
```

Comment:
The ordering of the modules in this file matters, and corresponds to the order in which the modules are concatenated together and thus ultimately the order in which initialization code is run in production.  So, we should be putting pointer.js where it belongs in that ordering.  I think just after `ui.js` would be a reasonable place for this, since it depends on code in `ui.js`  but code in `click_handlers.js` uses it.